docs/administration-guide/start.md

```diff
@@ -2,22 +2,33 @@
 
 Are you in charge of managing Metabase for your organization? Then you're in the right spot. You are the chosen one.
 
-**This guide will teach you about:**
```

Comment:
This re-ordering will muck with the "next" link at the bottom of each page.
Do we still want a linear path through the guide or should we just have a topic index?

Reply:
Yeah, I really would prefer to go to a navigable index, accessible from each article. But I'm concerned that that will be a time suck, so I'm thinking for now I might just fix each article's "next" link to follow the new order.